Example/watchOSSample/SampleWatchAppWatchKitExtension/ExtensionDelegate.swift

```diff
@@ -29,17 +30,25 @@ class ExtensionDelegate: NSObject, WKExtensionDelegate, MessagingDelegate {
       }
     }
     Messaging.messaging().delegate = self
+    let remoteConfig = RemoteConfig.remoteConfig()
+    remoteConfig.fetchAndActivate { status, error in
+      guard error == nil else {
+        print("error:" + error.debugDescription)
+        return
+      }
+      print("value:\n" + remoteConfig["test"].stringValue!)
```

Comment:
Nit: for those that aren't using a project with a `test` key, can you please change the force unwrap to a safe fallback? Something like `?? "No value for key 'test'"` or just `?? "nil"`

Reply:
Done.